e2e/_suites/fleet/features/system_integration.feature

```diff
@@ -189,22 +189,22 @@ Examples: Debian
 | os     |
 | debian |
 
-#@deploy-system_integration-with-filesystem
-#Scenario Outline: Adding the System Integration to an Policy
-#   Given a "<os>" agent is deployed to Fleet with "tar" installer
-#   And the agent is listed in Fleet as "online"
-#   When the policy is updated to have "system/metrics" set to "filesystem"
-#   Then "system/metrics" with "filesystem" metrics are present in the datastreams
-#
+@deploy-system_integration-with-filesystem
+Scenario Outline: Adding the System Integration to an Policy
+   Given a "<os>" agent is deployed to Fleet with "tar" installer
+   And the agent is listed in Fleet as "online"
+   When the policy is updated to have "system/metrics" set to "filesystem"
+   Then "system/metrics" with "filesystem" metrics are present in the datastreams
+
 #@centos
```

Comment:
Do we keep Centos as excluded?

Reply:
I ask because Debian tests are not run in a PR basis, but instead on merges and nightly schedules.

Reply:
I think centos fails here and we should probably figure out why rather than comment it out

Reply:
Centos is failing because the filesystem metrics does not show up in datastream after policy updated. we need to talk to agent guys on this problem

Reply:
There is no better price to a test framework that detecting a real bug 🎉 !